Compilation failure with Rust 1.31 on Windows

This crate targets backwards compatibility with Rust 1.31, but fails to compile with 1.31 under Windows. This is because it relies on `uniform_paths` which was not stabilised until 1.32.

```
error[E0658]: imports can only refer to extern crate names passed with `--extern` on stable channel (see issue #53130)
  --> C:\Users\runneradmin\.cargo\registry\src\github.com-1ecc6299db9ec823\terminal_size-0.1.16\src\lib.rs:34:9
   |
32 | mod windows;
   | ------------ not an extern crate passed with `--extern`
33 | #[cfg(windows)]
34 | pub use windows::{terminal_size, terminal_size_using_handle};
   |         ^^^^^^^
   |
note: this import refers to the module defined here
  --> C:\Users\runneradmin\.cargo\registry\src\github.com-1ecc6299db9ec823\terminal_size-0.1.16\src\lib.rs:32:1
   |
32 | mod windows;
   | ^^^^^^^^^^^^
```

The `pub use` path should be prefixed with `crate::` the same way it is for unix.

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,4 +1,4 @@
-//! A simple utility for getting the size of a terminal.  
+//! A simple utility for getting the size of a terminal.
 //!
 //! Supports both Linux, MacOS, and Windows.
 //!
@@ -31,7 +31,7 @@ pub use crate::unix::{terminal_size, terminal_size_using_fd};
 #[cfg(windows)]
 mod windows;
 #[cfg(windows)]
-pub use windows::{terminal_size, terminal_size_using_handle};
+pub use crate::windows::{terminal_size, terminal_size_using_handle};
 
 #[cfg(not(any(unix, windows)))]
 pub fn terminal_size() -> Option<(Width, Height)> { None }

